app/views/project_leads/index.html.erb

```diff
@@ -28,9 +28,9 @@
 </table>
 
 <div class='row'>
-  <div class='col-md-8 offset-md-4'>
-    <%# <%if current_user.roles.include?('Business_Developer') %>
+  <div class='col-md-8 offset-md-5'>
+    <%if current_user.roles.pluck(:name).include?('Business_Developer') %>
```

Comment:
bad. This should  not be here

Reply:
I called a function business_developer here but it was'nt working so had to do this. I need to ask this.

Reply:
Created Function in Model